### PR TITLE
Port over to ROS2 foxy

### PR DESCRIPTION
**Pull request by knatsuki**
Follow up for #44. Pardon for the delay! I can confirm the launch file is working on the latest foxy distro. Let me know if there is anything more I can do. Thanks!

### CODE REVIEW

File: rviz/urdf.rviz [hunk 1]
```diff
@@ -50,9 +44,17 @@ Visualization Manager:
       Plane Cell Count: 10
       Reference Frame: <Fixed Frame>
       Value: true
-    - Alpha: 0.5
-      Class: rviz/RobotModel
+    - Alpha: 1
```
Comment (DLu) [inline rviz/urdf.rviz]: Any particular reason for changing the alpha?

**Review by DLu — APPROVED** [review]

File: rviz/urdf.rviz (hunk 1 above)
Comment (knatsuki) [reply rviz/urdf.rviz]: None at all. It got unintentionally changed when I was tinkering with the config. I think the only critical changes are the configuration for loading the robot_description and the plugin names.

**Comment by DLu** [comment]
Thanks for porting this!

**Comment by knatsuki** [comment]
Not at all! Thanks for the quick follow up!

**Pull request merged**